app/controllers/env_controller.rb

```diff
@@ -0,0 +1,12 @@
+# Controller for the "environments status" page
+class EnvController < ApplicationController
```

Comment:
Looks like this controller has no test coverage. We should probably add a controller spec for it.

Reply:
Added a controller test in 5db53bd (we have the `DeployStatusChecker` test coverage in `/spec/services/deploy_status_checker_spec.rb` already).